doc/en/components/date-picker.md

```diff
@@ -0,0 +1,90 @@
+---
+title: $PlatformShort$ Date Picker | Data Visualization Tools | Infragistics
+_description: Infragistics' $PlatformShort$ date picker component helps your users select dates. Improve your graphs and visualization with Ignite UI for  $PlatformShort$!
+_keywords: $PlatformShort$ date picker, drop down, $ProductName$, Infragistics
+mentionedTypes: []
+---
+# $PlatformShort$ Date Picker Overview
+
+The Date Picker component allows users to use a drop-down calendar UI allowing the intuitive selection of a day, month and year. This can be helpful when an application user needs to select specific dates, and multiple editors can be combined to create a date-range UI.
+
+## Usage
+
+The $PlatformShort$ Date Picker editor component consists of three default parts - the text area, a calendar drop-down button, and a clear button. The text area will display the selected date, stored in the `Value` property of the editor.
+
```

Comment:
add bullets here to describe each area 
Text Area- the text area will display the selected date, stored in the value property of the editor
Calendar -
Clear button -